src/EditorFeatures/CSharpTest/CSharpEditorServicesTest.csproj

```diff
@@ -65,13 +65,15 @@
     <Reference Include="System.XML" />
     <Reference Include="System.Xml.Linq" />
     <Reference Include="WindowsBase" />
+    <!-- Force a reference to Microsoft.VisualStudio.Text.Internal. This NuGet package includes an empty ref folder so nobody accidentally uses APIs from it,
+         but includes the lib folder so it's copied as a dependency of Microsoft.VisualStudio.Platform.VSEditor. Unfortunately we still have test code that
+         does depend on internal APIs, so we need to a force a reference here. -->
+    <Reference Include="$(NuGetPackageRoot)\microsoft.visualstudio.text.internal\$(MicrosoftVisualStudioTextInternalVersion)\lib\net46\Microsoft.VisualStudio.Text.Internal.dll" />
```

Comment:
FYI @olegtk we're doing this hack now because of IWaitIndicator issues were we need to provide an implementation for your interface as well as our own.